src/tracing.ts

```diff
@@ -0,0 +1,34 @@
+import * as vscode from "vscode";
+import * as config from "./config";
+
+export enum messageType {
+  Info = "Info",
+  Warning = "Warning",
+  Error = "Error",
+  Verbose = "Verbose",
+  Debug = "Debug"
+}
+
+export class Trace {
+  private _outputChannel: any;
+  public messageType = messageType;
```

Comment:
Is this class member used by anything?

Reply:
This controls the message type, seend after the initial date/time; in the sample screenshot this is the "Info: " part. It would be nice to use an icon or a different color for warnings and errors, my intent here was to add something that we can expand upon (there may be other scenarios we want to add tracing to and may need to capture errors and so on)